test/test_unary_ufuncs.py

```diff
@@ -1219,6 +1219,26 @@ def check_equal(t):
         t = torch.tensor([min, max, eps, tiny], dtype=dtype, device=device)
         check_equal(t)
 
+    @dtypes(torch.float32, torch.float64)
+    @unittest.skipIf(not TEST_SCIPY, "SciPy not found")
+    def test_special_ndtr_vs_scipy(self, device, dtype):
+        def check_equal(t):
+            # Test by comparing to scipy
+            actual = torch.special.ndtr(t)
+            expected = scipy.special.ndtr(t.cpu().numpy())
+            self.assertEqual(actual, expected)
+
+        range = (-10, 10)
+
+        t = torch.linspace(*range, int(1e4), device=device, dtype=dtype)
+        check_equal(t)
+
+        # NaN, inf, -inf are tested in reference_numerics tests.
```

Comment:
Aren't they tested in reference_numerics_extremal?

Reply:
What I meant was that these values are already tested to so we don't need to test them. Instead we test other extremal values like `eps`, `min`, `max`, `tiny`.
Should I update the comment in case we do keep the test.